spec/models/participant_declaration_spec.rb

```diff
@@ -22,6 +23,18 @@
     it { is_expected.to validate_presence_of(:declaration_date) }
     it { is_expected.to validate_presence_of(:declaration_type) }
     it { is_expected.to validate_presence_of(:cohort) }
+
+    context "when voided_at is present" do
```

Comment:
is it worth adding a test that if both are `nil` that the declaration is valid?

Reply:
I'll add explicit tests to ensure the fields are not validated when nil 👍